README.md

```diff
@@ -119,6 +119,79 @@ Connection: keep-alive
 
 This error shows the request is missing a required property `name` from the HTTP request body.
 
+#### Server Validation
+
+You can simulate a request to a real server and verify whether it is valid according to the spec
+by providing a `__server` query param.
+
+Take this minimalistic spec (oas3) example:
+
+```json
+{
+  "openapi": "3.0.0",
+  "paths": {
+    "/pet": {
+      "get": {
+        "responses": {
+          "200": {
+            "content": {
+              "*/*": {
+                "schema": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "servers": [
+    {
+      "url": "{schema}://{host}/{basePath}",
+      "variables": {
+        "schema": {
+          "default": "http",
+          "enum": ["http", "https"]
+        },
+        "host": {
+          "default": "stoplight.io",
+          "enum": ["stoplight.io", "dev.stoplight.io"]
+        },
+        "basePath": {
+          "default": "api"
+        }
+      }
+    }
+  ]
+}
+```
+
+You can make a request enforcing server validation this way
+
+```bash
+curl -X GET localhost:4010/pet?__server=http://stoplight.io/api
+# will give you a lorem ipsum response, like so:
```

Comment:
These comments do not make it clear what I will be seeing if this runs successfully. Why are we talking about lorum ipsum? Maybe we could just say "It will give you a mocked response as usual" without getting into dynamic or static or any other sidepoints?